docs/embrace-api/grafana_integrations.md

```diff
@@ -14,11 +14,15 @@ sidebar_position: 1
 ## Setting Up Embrace as a Data Source
 
 Follow these steps to add Prometheus as a data source in the Grafana dashboard:
-1. Click the gear icon to go to the Configurations page.
```

Comment:
This was wrong. So I decided to fix it.

Reply:
I'm struggling with Grafana != Grafana Cloud.  But this was too clunky.

Reply:
@scottieb3 do you prefer Grafana or Grafana Cloud? I can do the change to have consistency.